Index: mlir/include/mlir/IR/FunctionInterfaces.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mlir/include/mlir/IR/FunctionInterfaces.h b/mlir/include/mlir/IR/FunctionInterfaces.h
--- a/mlir/include/mlir/IR/FunctionInterfaces.h	(revision e1318078a4e160eb723bcbcfcdcc9a1b618f7067)
+++ b/mlir/include/mlir/IR/FunctionInterfaces.h	(date 1651883503354)
@@ -235,15 +235,15 @@
       // Verify that all of the argument attributes are dialect attributes, i.e.
       // that they contain a dialect prefix in their name.  Call the dialect, if
       // registered, to verify the attributes themselves.
-      for (auto attr : argAttrs) {
-        if (!attr.getName().strref().contains('.'))
-          return op.emitOpError("arguments may only have dialect attributes");
-        if (Dialect *dialect = attr.getNameDialect()) {
-          if (failed(dialect->verifyRegionArgAttribute(op, /*regionIndex=*/0,
-                                                       /*argIndex=*/i, attr)))
-            return failure();
-        }
-      }
+//      for (auto attr : argAttrs) {
+//        if (!attr.getName().strref().contains('.'))
+//          return op.emitOpError("arguments may only have dialect attributes");
+//        if (Dialect *dialect = attr.getNameDialect()) {
+//          if (failed(dialect->verifyRegionArgAttribute(op, /*regionIndex=*/0,
+//                                                       /*argIndex=*/i, attr)))
+//            return failure();
+//        }
+//      }
     }
   }
   if (ArrayAttr allResultAttrs = op.getAllResultAttrs()) {
@@ -267,16 +267,16 @@
       // Verify that all of the result attributes are dialect attributes, i.e.
       // that they contain a dialect prefix in their name.  Call the dialect, if
       // registered, to verify the attributes themselves.
-      for (auto attr : resultAttrs) {
-        if (!attr.getName().strref().contains('.'))
-          return op.emitOpError("results may only have dialect attributes");
-        if (Dialect *dialect = attr.getNameDialect()) {
-          if (failed(dialect->verifyRegionResultAttribute(op, /*regionIndex=*/0,
-                                                          /*resultIndex=*/i,
-                                                          attr)))
-            return failure();
-        }
-      }
+//      for (auto attr : resultAttrs) {
+//        if (!attr.getName().strref().contains('.'))
+//          return op.emitOpError("results may only have dialect attributes");
+//        if (Dialect *dialect = attr.getNameDialect()) {
+//          if (failed(dialect->verifyRegionResultAttribute(op, /*regionIndex=*/0,
+//                                                          /*resultIndex=*/i,
+//                                                          attr)))
+//            return failure();
+//        }
+//      }
     }
   }

